apps/state/test/state/server_test.exs

```diff
@@ -251,7 +251,7 @@ defmodule State.ServerTest do
         %Example{id: 2, data: :other}
       ])
 
-      assert [%{data: 38}, %{data: 44}, %{data: :other}] = HooksServer.all()
+      assert [%{data: :other}, %{data: 38}, %{data: 44}] = HooksServer.all()
```

Comment:
@paulswartz got a bit of a stumper here. With this update, my Mac wants this list to be in this order, while the Linux GHA runner wants it to be in the old order. I did some digging around in the OTP repo and couldn't find anything obvious that introduced this, but it seems like it might be a bug? Closed thing I found was this, which suggests the behavior observed on Mac is a regression: https://github.com/erlang/otp/pull/6752.